app/controllers/collection_drafts_controller.rb

```diff
@@ -89,7 +59,11 @@ def create
       end
     else # record update failed
       flash[:error] = I18n.t("controllers.draft.#{plural_resource_name}.create.flash.error")
+      load_umm_schema
```

Comment:
render loads the view without calling the function.  This means that many of the variables that are set in the new function are set, so they need to be set now.  render is used so that the data the user did change can be retained.